test/integration/extension_discovery_integration_test.cc

```diff
@@ -857,6 +857,7 @@ TEST_P(ExtensionDiscoveryIntegrationTest, ReloadBoth) {
 
 // ECDS config dump test with one listener ECDS filter and one HTTP ECDS filter.
 TEST_P(ExtensionDiscoveryIntegrationTest, ConfigDumpWithTwoSubscriptionTypes) {
+  DISABLE_IF_ADMIN_DISABLED; // Uses admin interface.
```

Comment:
With https://github.com/envoyproxy/envoy/pull/23903, the ADMIN interface is disabled in compile_time_options CI build. Have to disable these tests in this case since config_dump requires admin interface.